### PR TITLE
Update `README.md` boilerplate from skeleton

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request updates some boilerplate language from the upstream skeleton in `README.md`.

## 💭 Motivation and context ##

This should have been done when the repo was created but for some reason it was not.

## 🧪 Testing ##

:eyes:

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,9 @@
...
 [![GitHub Build Status](https://github.com/cisagov/terraformer-packer/workflows/build/badge.svg)](https://github.com/cisagov/terraformer-packer/actions)
 
-This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) GitHub
-[Packer](https://packer.io) project started. This skeleton project
-contains [licensing information](LICENSE), as well as
-[pre-commit hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for the major languages that we use.
+This project can be user to create Amazon Machine Images (AMIs) that
```
Comment (dav3r) [inline README.md:5]: Fix typo:
```suggestion
This project can be used to create Amazon Machine Images (AMIs) that
```

**Review by dav3r — APPROVED** [review]
👍

**Review by dv4harr10 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -2,13 +2,9 @@
...
-This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) GitHub
-[Packer](https://packer.io) project started. This skeleton project
-contains [licensing information](LICENSE), as well as
-[pre-commit hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for the major languages that we use.
+This project can be used to create Amazon Machine Images (AMIs) that
+include the [Terraform](https://www.terraform.io/) infrastructure as
+code (IaC) tool from [HashiCorp](https://www.hashicorp.com/).
```
Comment (mcdonnnj) [inline README.md]: I think it's also worth mentioning that Ansible is installed and available as well.

**Review by mcdonnnj — APPROVED** [review]
LGTM ✔ One minor suggestion for your consideration.

File: README.md (hunk 2 above)
Comment (jsf9k) [reply README.md]: Please see commit 58ddc29.

Comment (mcdonnnj) [reply README.md]: Thanks!

**Pull request merged**